### DISCUSSION TITLE
Branches not loading when specifying new build

### DISCUSSION THREAD

**Discussion by rseigel** [codemagic-ci-cd/codemagic-docs#1302] [Q&A] [answered]
Hi there. Out of nowhere we are unable to create a new build because we cannot specify the branch. The branch list just hangs. I've attached a screenshot. Please let me know what information you need to help debug.

Thanks!
Rob
<img width="907" alt="Screenshot 2023-01-09 at 11 11 47 AM" src="https://user-images.githubusercontent.com/10031333/211378322-e86045ae-2fe2-4a06-8b96-bf6614421b06.png">

**Comment by icarusdust** [comment, marked as answer]
Hi @rseigel, please try disconnecting your Gitlab integration (or any other) in the team/user settings and connect it back. Let me know if it helps.

**Comment by rseigel** [comment]
Hi @icarusdust. That worked! Thanks so much.

Rob
